_includes/activity/github-desktop/02-clone-locally.md

```diff
@@ -4,5 +4,5 @@ After you've created a repository on the remote, the next step is to clone it to
 1. On GitHub.com, navigate to the Code tab of the repository.
 1. On the right side of the screen, click `Clone or download`.
```

Comment:
@hectorsector I saw that, but the Open in Desktop is one of the cool features and I thought we could create the instructions so they are correct and then re-do the GIF after `@brianamarie` gets back from OOO.